android/java/org/chromium/chrome/browser/BraveRewardsUserWalletActivity.java

```diff
@@ -83,7 +83,7 @@ private void SetUIControls() {
             SetBtnOpenUrlClickHandler(
                     btn2, intent.getStringExtra(BraveRewardsExternalWallet.WITHDRAW_URL));
         } else {
-            // CONNECTED or PENDING
+            // CONNECTED
```

Comment:
This shouldn't say `CONNECTED` anymore, as this `CONNECTED` represents the old `CONNECTED` state (`mojom::WalletStatus::CONNECTED (1)`). Here the only value `status` can hold is `WalletStatus.LOGGED_OUT`.

Reply:
Let me correct this one as well: so `status` here can only hold the values `WalletStatus.NOT_CONNECTED`, or `WalletStatus.LOGGED_OUT`.